### PR TITLE
bootloader: fix GNOME-Next and Krypton at booting

### PR DESCRIPTION
**Pull request by nilxam**
GNOME-Next and Krypton's version is not Tumbleweed thus previous change has not make GNOME-Next and Krypton tests success to boot. Get rid of specific architecture check and also without Tumbleweed version check, the boot key on livecd will be decided by LIVECD_LOADER variable exists or not.

Related ticket: https://progress.opensuse.org/issues/53993
Verification run
GNOME-Live.i686 : http://10.163.1.11/tests/81
GNOME-Live.x86_64 : http://10.163.1.11/tests/80
Standard xfce installation with normal TW iso: http://10.163.1.11/tests/77
Krypton-live: http://10.163.1.11/tests/79
GNOME-Next: http://10.163.1.11/tests/76

### CODE REVIEW

File: tests/installation/bootloader.pm [hunk 1]
```diff
@@ -24,17 +24,21 @@ use bootloader_spvm;
...
     return           if select_bootmenu_option == 3;
+    # the default loader is isolinux on openSUSE/SLE products with product-builder
     my $boot_cmd = 'ret';
     # Tumbleweed livecd has been switched to grub with kiwi 9.17.41 except 32bit
-    uefi_bootmenu_params() if (is_livecd && is_tumbleweed && !is_i586 && !is_i686);
+    # when LIVECD_LOADER has set grub2 then change the boot command for grub2
+    if (is_livecd && check_var('LIVECD_LOADER', 'grub2')) {
+        $boot_cmd = 'ctrl-x';
+    }
+    uefi_bootmenu_params() if (is_livecd && check_var('LIVECD_LOADER', 'grub2'));
```
Comment (DimStar77) [inline tests/installation/bootloader.pm]: The if-condition seems to be exactly the same as the if block just above - why not take it into the if block?

Comment (nilxam) [reply tests/installation/bootloader.pm]: yeah, idk why I didn't do that. :) Thanks.

**Comment by Vogtinator** [comment]
LGTM. I added `LIVECD_LOADER=grub2` to all affected medium types on o3 and removed some obsolete variables while at it.

**Comment by okurz** [comment]
As you stated that you configured all media accordingly this should be fine. An alternative would be to encode this into e.g. products/opensuse/main.pm but everything is possible :)

**Pull request merged**